src/javascripts/components/tabs.js

```diff
@@ -5,20 +5,23 @@
 
   GOVUK.tabs = {
 
-    // Tabs mode
-    clickTabItem: function (e) {
-      e.preventDefault()
-
-      // Get tab container ID
-      var id = $(this).attr('href')
+    // Reset tabs and containers
+    resetTabs: function (id) {
+      var $example = $(id).parent()
 
       // Reset state
-      $('.app-c-tabs__item').removeClass('app-c-tabs__item--current')
-      $('.app-c-tabs__heading').removeClass('app-c-tabs__heading--current')
-      $('.app-c-tabs__item a').removeAttr('aria-selected')
-      $('.app-c-tabs__container').hide().attr('aria-hidden', 'true')
+      $example.find('.app-c-tabs__item').removeClass('app-c-tabs__item--current')
+      $example.find('.app-c-tabs__heading').removeClass('app-c-tabs__heading--current')
+      $example.find('.app-c-tabs__item a').removeAttr('aria-selected')
+      $example.find('.app-c-tabs__container').hide().attr('aria-hidden', 'true')
```

Comment:
I would prefer to use a class that hides or shows the container to separate the presentation from this but this works too.